contessa/__init__.py

```diff
@@ -6,6 +6,8 @@
 check on load in temporary table.
 """
 
+__version__ = "0.1.5"
```

Comment:
can we keep the version 0.1.4 ? we can release later. I wanted to done at least https://github.com/kiwicom/contessa/issues/57 one